docs/INGRESS.md

```diff
@@ -0,0 +1,46 @@
+# Ingress
+
+All ingress is handled by the [nginx ingress controller](https://kubernetes.github.io/ingress-nginx/) which is maintained under the [kubernetes github organization](https://github.com/kubernetes/ingress-nginx/). By default traffic that does not match any ingress will receive the mvpstudio.org homepage.
```

Comment:
Big fan of wrapping README's to 120 characters or less so you can read them from the command line via `less` or `cat`.

Reply:
Will update